File: C/Listas/Lista 01/ex12.c
//Calcule o valor de uma prestação em atraso, utilizando a fórmula: PRESTAÇÃO = VALOR + (VALOR * (TAXA/100) * TEMPO).
#include <stdio.h>
#include <stdlib.h>

int main(){
    int valor, taxa, tempo, prest;
    printf("Por favor, insira o valor da sua prestaçâo em atraso: R$");
    scanf("%d", &valor);
    printf("Agora, insira a taxa da prestação: ");
    scanf("%d", &taxa);
    printf("Insira em quantos meses a prestação será paga: ");
    scanf("%d", &tempo);
    prest=valor+(valor*taxa/100*tempo);

    printf("De acordo com os valores informados sua prestação sairá no total de R$%d", prest);
    return 0;
}

File: Java/extra/Lista 01/src/ex12.java
/*Calcule o valor de uma prestação em atraso, utilizando a fórmula: PRESTAÇÃO = 
VALOR + (VALOR * (TAXA/100) * TEMPO). */

import java.util.Scanner;

public class ex12 {
    static double prestacao(double a, int b, int c) {
        double x = a + (a * ((double) b / 100) * (double) c);
        return x;
    }

    public static void main(String[] args) {
        int taxa, tempo;
        double valor = 0;
        System.out.println("Digite o valor em atraso: ");
        try (Scanner leitor = new Scanner(System.in)) {
            valor = leitor.nextDouble();
            System.out.println("\nDigite a porcentagem da taxa: ");
            taxa = leitor.nextInt();
            System.out.println("\nDigite o tempo da prestação em meses: ");
            tempo = leitor.nextInt();
        }

        System.out.format("\nA sua prestação será de R$%.2f", prestacao(valor, taxa, tempo));
    }
}
